wger/exercises/views/exercises.py

```diff
@@ -133,8 +133,11 @@ def view(request, id, slug=None):
     template_data['show_shariff'] = True
 
     exercise = get_object_or_404(Exercise, pk=id)
+    variations = Exercise.objects.select_related().filter(
+        variations=exercise.variations).exclude(name=exercise.name)
```

Comment:
Since we will probably need the variations in other places, I would suggest moving this filter logic to the exercises model, perhaps a property like this?

```python
@property
def get_variations(self):
    return self.variations.exercise_set.exclude(id=self.id)
```

then we can just access it in the template